composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "ab52f12570440f5833f0d81bce06f884",
+    "content-hash": "0a58509a75dc4ea99a4bef3d13ee4075",
```

Comment:
A heads up, `composer.json` and `.lock` has changed since this was originally created, the merge is pretty ugly.